README.adoc

```diff
@@ -0,0 +1,152 @@
+= Istio Circuit Breaker mission
+
+== Purpose
+Showcase Istio's Circuit Breaker via a (minimally) instrumented Spring Boot application
+
+== Prerequisites
+. Openshift 3.9 cluster
+. Istio 0.7.1 with authentication installed on the aforementioned cluster. To install Istio simply follow one of the following docs:
+.. https://istio.io/docs/setup/kubernetes/quick-start.html
+.. https://istio.io/docs/setup/kubernetes/ansible-install.html
+. Enable automatic sidecar injection for Istio (See https://istio.io/docs/setup/kubernetes/sidecar-injection.html[this] for details)
++
+In order for Istio automatic sidecar injection to work properly, the following Istio configuration needs to be in place:
++
+.. The `policy` field is set to `disabled` in the `istio-inject` configmap  of the `istio-system` namespace.
+.. The `istio-sidecar-injector` `MutatingWebhookConfiguration` should not limit the injection to properly labeled namespaces
+
+. Expose services and Istio ingress:
```

Comment:
This is still unneeded. There is no need to expose the `istio-ingress` for this booster.